gvfs-helper-client.c

```diff
@@ -337,17 +338,25 @@ static struct gh_server__process *gh_client__find_long_running_process(
 	struct gh_server__process *entry;
 	struct strvec argv = STRVEC_INIT;
 	struct strbuf quoted = STRBUF_INIT;
+	int fallback;
 
 	gh_client__choose_odb();
 
 	/*
 	 * TODO decide what defaults we want.
 	 */
 	strvec_push(&argv, "gvfs-helper");
-	strvec_push(&argv, "--fallback");
 	strvec_push(&argv, "--cache-server=trust");
 	strvec_pushf(&argv, "--shared-cache=%s",
 			 gh_client__chosen_odb->path);
+
+	/* If gvfs.fallback=false, then don't add --fallback. */
```

Comment:
Would it be easier to just hack gvfs-helper.exe to lookup that config value ??

Reply:
Might be too magic, though. And so far, `gvfs-helper` _only_ uses [`git_config(git_default_config)`](https://github.com/microsoft/git/blob/vfs-2.45.2/gvfs-helper.c#L4212), no custom config variables there.

Reply:
d'oh, nevermind.

Reply:
Moreover, the client passes `--fallback` to the server directly, which would normally override any config option.